website/client/src/components/groups/group.vue

```diff
@@ -498,7 +498,8 @@ export default {
     if (!this.searchId) this.searchId = this.groupId;
     await this.fetchGuild();
     this.$store.dispatch('common:setTitle', {
-      section: this.group.name,
+      section: this.t$('groupPlans'),
+      subSection:this.group.name,
```

Comment:
I think lint is going to complain about the lack of space before this.group.name